### PR TITLE
Release notes v0.36 — BK section

### PR DESCRIPTION
**Pull request by isaacdevlugt**

### CODE REVIEW

File: doc/releases/changelog-dev.md [hunk 1]
```diff
@@ -96,27 +95,39 @@
...
-* The `qml.qchem.hf_state` function is upgraded to be compatible with the parity and Bravyi-Kitaev bases.
+* The `qml.qchem.hf_state` function has been upgraded to be compatible with
+  `qml.fermi.parity_transform` and the new Bravyi-Kitaev mapping 
+  (`qml.fermi.bravyi_kitaev`).
   [(#5472)](https://github.com/PennyLaneAI/pennylane/pull/5472)
 
+  ```pycon
+  >>> state_bk = qml.qchem.hf_state(2, 6, basis="bravyi_kitaev")
+  >>> print(state_bk)
+  [1 0 0 0 0 0]
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:125]: 🍔 👑

File: doc/releases/changelog-dev.md [hunk 2]
```diff
@@ -96,27 +95,39 @@
 
   ```
 
-<h4>Make use of more methods to map from molecules 🗺️</h4>
-
+<h4>More methods to map molecules to qubits 🗺️</h4>
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:98]: Very minor, but I had a soft spot for the alliteration and use of a very to start each section.

File: doc/releases/changelog-dev.md [hunk 3]
```diff
@@ -96,27 +95,39 @@
...
 
-* Added new function `qml.bravyi_kitaev` to map fermionic Hamiltonians to qubit Hamiltonians.
+* A new function called `qml.fermi.bravyi_kitaev` has been added to map fermionic
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:100]:
```suggestion
* A new function called `qml.bravyi_kitaev` has been added to map fermionic 
```
Prefer to link to the top-level function (it's also the one in the [docs](https://xanaduai-pennylane--5575.com.readthedocs.build/en/5575/code/api/pennylane.bravyi_kitaev.html))

File: doc/releases/changelog-dev.md [hunk 4]
```diff
@@ -96,27 +95,39 @@
...
 
-  ```python
-  import pennylane as qml
-  fermi_ham = qml.fermi.from_string('0+ 1-')
-
-  qubit_ham = qml.bravyi_kitaev(fermi_ham, n=6)
-  ```
+  In addition to `qml.fermi.jordan_wigner` and `qml.fermi.parity_transform`, the
+  Bravyi-Kitaev mapping is available via `qml.fermi.bravyi_kitaev` to map
+  fermionic Hamiltonians to qubit Hamiltonians. Simply provide a fermionic
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:106]:
```suggestion
  This function presents an alternative mapping to `qml.jordan_wigner` or
  `qml.parity_transform`. Simply provide a fermionic 
```
Just felt like this sentence repeated the first one a bit.

File: doc/releases/changelog-dev.md [hunk 5]
```diff
@@ -96,27 +95,39 @@
...
+  Bravyi-Kitaev mapping is available via `qml.fermi.bravyi_kitaev` to map
+  fermionic Hamiltonians to qubit Hamiltonians. Simply provide a fermionic 
+  Hamiltonian (created from `from_string`, `FermiA`, `FermiC`, `FermiSentence`, 
+  or `FermiWord`) and the number of qubits / spin orbitals in the system, `n`:
 
   ```pycon
+  >>> fermi_ham = qml.fermi.from_string('0+ 1-')
+  >>> qubit_ham = qml.bravyi_kitaev(fermi_ham, n=6)
   >>> print(qubit_ham)
   -0.25j * Y(0.0) + (-0.25+0j) * X(0) @ Z(1.0) + (0.25+0j) * X(0.0) + 0.25j * Y(0) @ Z(1.0)
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:114]: Would update these lines:
```
>>> qml.bravyi_kitaev(fermi_ham, n=6)
(
    -0.25j * Y(0)
  + (-0.25+0j) * (X(0) @ Z(1))
  + (0.25+0j) * X(0)
  + 0.25j * (Y(0) @ Z(1))
)
```
@soranjh @ddhawan11 would it be challenging to cast the coeffs to real? Do we ever expect them to be complex?

File: doc/releases/changelog-dev.md [hunk 6]
```diff
@@ -96,27 +95,39 @@
...
   ```
 
-* The `qml.qchem.hf_state` function is upgraded to be compatible with the parity and Bravyi-Kitaev bases.
+* The `qml.qchem.hf_state` function has been upgraded to be compatible with
+  `qml.fermi.parity_transform` and the new Bravyi-Kitaev mapping
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:118]:
```suggestion
  `qml.parity_transform` and the new Bravyi-Kitaev mapping 
```

File: doc/releases/changelog-dev.md [hunk 7]
```diff
@@ -96,27 +95,39 @@
...
+  >>> fermi_ham = qml.fermi.from_string('0+ 1-')
+  >>> qubit_ham = qml.bravyi_kitaev(fermi_ham, n=6)
   >>> print(qubit_ham)
   -0.25j * Y(0.0) + (-0.25+0j) * X(0) @ Z(1.0) + (0.25+0j) * X(0.0) + 0.25j * Y(0) @ Z(1.0)
   ```
 
-* The `qml.qchem.hf_state` function is upgraded to be compatible with the parity and Bravyi-Kitaev bases.
+* The `qml.qchem.hf_state` function has been upgraded to be compatible with
+  `qml.fermi.parity_transform` and the new Bravyi-Kitaev mapping 
+  (`qml.fermi.bravyi_kitaev`).
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:119]:
```suggestion
  (`qml.bravyi_kitaev`).
```
Maybe the brackets are not needed? :thinking:

File: doc/releases/changelog-dev.md [hunk 8]
```diff
@@ -96,27 +95,39 @@
...
   [(#5472)](https://github.com/PennyLaneAI/pennylane/pull/5472)
 
+  ```pycon
+  >>> state_bk = qml.qchem.hf_state(2, 6, basis="bravyi_kitaev")
+  >>> print(state_bk)
+  [1 0 0 0 0 0]
+  >>> state_parity = hf_state(2, 6, basis="parity")
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:126]:
```suggestion
  >>> state_parity = qml.qchem.hf_state(2, 6, basis="parity")
```

**Review by trbromley — COMMENTED** [review]
Thanks @isaacdevlugt! Just left a few comments.

File: doc/releases/changelog-dev.md (hunk 5 above)
Comment (soranjh) [reply doc/releases/changelog-dev.md:114]: @trbromley In this simple example, `0.25j` is a large number to neglect and cast to real. But we can give you a bit larger examples where the coeffs have negligible imaginary part. Is that what you want?

File: doc/releases/changelog-dev.md [hunk 9]
```diff
@@ -96,27 +95,39 @@
...
+  efficiently on hardware. Simply provide a fermionic 
+  Hamiltonian (created from `from_string`, `FermiA`, `FermiC`, `FermiSentence`, 
+  or `FermiWord`) and the number of qubits / spin orbitals in the system, `n`:
 
   ```pycon
+  >>> fermi_ham = qml.fermi.from_string('0+ 1+ 1- 0-')
+  >>> qubit_ham = qml.bravyi_kitaev(fermi_ham, n=6, tol=0.0)
   >>> print(qubit_ham)
-  -0.25j * Y(0.0) + (-0.25+0j) * X(0) @ Z(1.0) + (0.25+0j) * X(0.0) + 0.25j * Y(0) @ Z(1.0)
+  0.25 * I(0) + -0.25 * Z(0) + -0.25 * (Z(0) @ Z(1)) + 0.25 * Z(1)
```
Comment (isaacdevlugt) [inline doc/releases/changelog-dev.md:114]: @trbromley updated!

Comment (trbromley) [reply doc/releases/changelog-dev.md:114]: Nice!!

**Review by trbromley — APPROVED** [review]
Thanks @isaacdevlugt!

File: doc/releases/changelog-0.36.0.md [hunk 10]
```diff
@@ -125,23 +125,37 @@
 
 <h4>Make use of more methods to map from molecules 🗺️</h4>
 
-* Added new function `qml.bravyi_kitaev` to map fermionic Hamiltonians to qubit Hamiltonians.
+* A new function called `qml.bravyi_kitaev` has been added to perform the
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.36.0.md:128]: @trbromley just resolved the merge conflicts but should be good now!

**Pull request merged**